Packs/KnowBe4KMSAT/Integrations/KnowBe4KMSAT/KnowBe4KMSAT_description.md

```diff
@@ -0,0 +1,6 @@
+## KnowBe4 KMSAT API help
+
+Integrate KnowBe4 and Palo Alto Networks' XSOAR platform for threat intelligence based on phishing tests and awareness training. Act on risk and automate remediation & more.
+
+### Generate API Key
+
```

Comment:
@kaleybeck Can you please address this?

Reply:
Usually we provide instructions for the customers who use the integration regarding on how to get the API key. If generating the API key is done using the product platform then the instructions should be provided, otherwise If the API key is not generated by the customer, you can remove the "### Generate API Key" line from this md file.